### DISCUSSION TITLE
[Suggestion/Request] Export Transactions button

### DISCUSSION THREAD

**Discussion by forReason** [Chia-Network/chia-blockchain#10443] [Ideas]
Feature request:
Add a button in order to export Transactions to csv sheet.
![image](https://user-images.githubusercontent.com/12736950/155820290-e4c99637-b88d-4e1b-ae57-aed7ef3f1a5e.png)

Ideally, when pressing the button, a drop down menu or something appears to select the date range.

**Comment by josea** [comment]
Note: make sure that the exported transactions actually add up to the balance (in my wallet if I copy/paste the transactions from the screen they do not add up).

**Comment by weeedy** [comment]
are there any news? or a command, so that i can do it by myself? thanks!

**Comment by wallentx** [comment]
I believe that this would be useful, especially for tax reasons. What items would you like to see on the export?
Using the CLI, you could currently retrieve all of the information available via the `chia wallet get_transactions`, or the `chia rpc wallet get_transactions` commands, but you'd have to add a few specific options and likely use `jq` to transform the data into a structure that would provide the data in an order that is usable, that you could then convert into a .csv.
